console/frontend/src/main/frontend/README.md

```diff
@@ -0,0 +1,27 @@
+# IafAngularNgupgrade
```

Comment:
```suggestion
# FfAngularNgUpgrade
```

Reply:
Op zich, ja, maar is eigenlijk alleen in de 8.0 nodig.